lib/mage/Mage.js

```diff
@@ -39,6 +39,9 @@ function testEngineVersion(pkg, pkgPath, mage) {
  */
 
 function Mage(mageRootModule) {
+	// Set up the worker id, if necessary
```

Comment:
I don't see any if-statements, so not entirely sure what "if necessary" is talking about?

Reply:
https://github.com/mage/mage/pull/151/files#diff-490b45026160e1275d20c3b8f66ab6b4R3

The function itself has conditional logic in it.

Reply:
Removed the comment overall.